modules/core/src/main/java/org/apache/synapse/deployers/InboundEndpointDeployer.java

```diff
@@ -185,4 +191,19 @@ public void restoreSynapseArtifact(String artifactName) {
                     "Restoring of the Inbound Endpoint named '" + artifactName + "' has failed", e);
         }
     }
+
+    private void callSynapseHandler(String inboundEndpointName, String startTime) throws DeploymentException {
```

Comment:
Shall we rename this to something like, executeAbstractExtendedSynapseHandler or something better to reflect what it does.

Reply:
Fixed